checks/lang.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+INVALID_LANG = %q(The LANG envrionment variable should not be set to C)
+
+def error_exit(message, code)
+  $stderr.puts message
+  exit code
+end
+
+begin
+    error_exit(INVALID_LANG, 1)  if ENV["LANG"] == "C"
+rescue
+    # Eat the exception and continue
```

Comment:
Likely none, but this seemed to be the pattern for the other checks. I assumed the logic was these should never fail